controllers/aliases.js

```diff
@@ -87,18 +87,27 @@ const aliases = {
     '/funding/awards-for-all': '/funding/under10k',
     '/funding/Awards-For-All': '/funding/under10k',
     '/funding/funding-finder?sc=1': '/funding/programmes/all',
+
+    // Older Managing Your Funding section
+    '/funding/funding-guidance/managing-your-funding': '/funding/managing-your-grant/promoting-your-project',
+    '/funding/funding-guidance/managing-your-funding/social-media': '/funding/managing-your-grant/promoting-your-project/social-media',
+    '/funding/funding-guidance/managing-your-funding/press': '/funding/managing-your-grant/promoting-your-project/your-local-press',
+    '/funding/funding-guidance/managing-your-funding/grant-acknowledgement-and-logos': '/funding/managing-your-grant/promoting-your-project/download-our-logo',
+    '/funding/funding-guidance/managing-your-funding/ordering-free-materials': '/funding/managing-your-grant/promoting-your-project/order-free-materials',
+    '/funding/funding-guidance/managing-your-funding/data-and-evidence': '/funding/managing-your-grant/gathering-evidence-and-learning/data-and-evidence',
```

Comment:
@scottoakley the above URL paths are before/after redirects – I renamed some of your new pages to have shorter URLs (eg. the second ones on each line after the colon). Do these look alright to you?!

Reply:
Looks good to me, thanks @mattandrews!